src/main/java/com/cloudbees/jenkins/plugins/sshagent/RemoteAgent.java

```diff
@@ -1,60 +0,0 @@
-/*
- * The MIT License
- *
- * Copyright (c) 2012, CloudBees, Inc., Stephen Connolly.
- *
- * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated documentation files (the "Software"), to deal
- * in the Software without restriction, including without limitation the rights
- * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
- *
- * The above copyright notice and this permission notice shall be included in
- * all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- * THE SOFTWARE.
- */
-
-package com.cloudbees.jenkins.plugins.sshagent;
-
-import hudson.model.TaskListener;
-
-import java.io.IOException;
-
-/**
- * Need an interface in order to export the object from the channel.
```

Comment:
Not sure if this was actually true prior to #48, but it is certainly not true now—this interface was not being passed to `Channel.export`.